examples/gno.land/p/demo/avl/list/list.gno

```diff
@@ -312,3 +314,19 @@ func (l *List) DeleteRange(startIndex, endIndex int) int {
 
 	return len(keysToDelete)
 }
+
+// ReadOnlyTree returns a read-only pointer to the underlying AVL tree.
+//
+// Example:
+//
+//	var l list.List
+//	l.Append(1, 2, 3)
+//
+//	rotree := l.ReadOnlyTree()
+//	rotree.ReverseIterateByOffset(0, rotree.Size(), func(key string, value any) bool {
+//	  println(value) // Output: 3 2 1
+//	  return false
+//	})
+func (l *List) ReadOnlyTree() *rotree.ReadOnlyTree {
```

Comment:
Let's keep avl.Tree as small as possible and avoid dependencies on other libraries. We can create another library that facilitates this, possibly using a type alias.